src/VolleyManagement.Backend/VolleyManagement.Crosscutting.Contracts/FeatureToggles/Core/DebugMode.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace VolleyManagement.Crosscutting.Contracts.FeatureToggles.Core
+{
+    using FeatureToggle.Core;
+    using Microsoft.Extensions.Configuration;
+
+    public class DebugMode : IFeatureToggle
```

Comment:
Add description to communicate it's meaning.

Have you tried inheriting from SimpleFeatureToggle to avoid parsing of appsettings directly?

Reply:
Yes, I tried. Did everything according to documentation but receiving error: "FeatureToggle.Core.ToggleConfigurationError: 'The key 'FeatureToggle.IisDeployment' was not found in AppSettings'"